src/main/java/olya/app/remindme/controller/EventController.java

```diff
@@ -36,4 +35,10 @@ List<EventResponseDto> get (@AuthenticationPrincipal User user) {
         List<Event> events = eventService.getAll(user.getId());
         return events.stream().map(eventMapper::mapToDto).collect(Collectors.toList());
     }
+    @GetMapping("/{id}/status")
```

Comment:
Nitpick: add a verb that describes the endpoint as it's not only getting the status of a event
Example: `/{id}/-update-status`